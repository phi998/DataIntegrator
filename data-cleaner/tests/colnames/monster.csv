Advertisement Title,Role,Location,Advertisement Content,Company Name,Salary,Advertisement Content
