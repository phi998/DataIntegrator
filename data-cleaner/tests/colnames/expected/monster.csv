Advertisement Title,Role,Location,Advertisement Content,Company Name,Employees,Links
